addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -74,7 +74,7 @@
 
                         <field name="state" readonly="1" widget="statusbar"/>
                     </header>
-                    <div class="alert alert-info my-0 mx-3 text-center" role="alert"
+                    <div class="alert alert-info m-0 text-center" role="alert"
```

Comment:
I'd add a `mb-2` ... or maybe just remove `m-0` since alerts have a margin-bottom by default, what do you think?